Dropdown/Dropdown.module.less

```diff
@@ -0,0 +1,39 @@
+// Dropdown.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.Dropdown {
+	.applySkins({
+		.item {
```

Comment:
Do we need to override the item styles or should we just keep the default item styles?

Reply:
I have added the style on the first item for border. we won't be able to use the default item style as don't have default border on the item.